spec/lib/guardian_spec.rb

```diff
@@ -2773,6 +2773,10 @@
       expect(Guardian.new(user).can_anonymize_user?(user)).to be_falsey
     end
 
+    it "it false for an annoymized user" do
```

Comment:
```suggestion
    it "it false for an anonymized user" do
```

Let's not annoy any users 🤪